.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "dotnet-ef": {
+      "version": "3.1.1",
+      "commands": [
+        "dotnet-ef"
+      ]
+    }
+  }
+}
```

Comment:
Please add a .gitignore file to ignore these system files like this and all of the files in the bin folder below which were added.  check here for some common .gitignore files https://github.com/github/gitignore

Reply:
I was thinking of adding a gitignore.  but I wasn't sure

Reply:
i think this file needs to be ignored still